cns/NetworkContainerContract.go

```diff
@@ -73,6 +74,7 @@ const (
 // CreateNetworkContainerRequest specifies request to create a network container or network isolation boundary.
 type CreateNetworkContainerRequest struct {
 	Version                    string
+	NCVersion                  string
```

Comment:
We should define it as an int

Reply:
Note: we need to persist the ProgrammedVersion so that we can avoid calling NMAgent again if CNS crashes

Reply:
1. I'll remove NCVersion. No need to record ProgrammedVersion in CreateNetworkContainerRequest since similar info got record in containerstatus and containerstatus include CreateNetworkContainerRequest. I added previously since the usage of Version is not clear in CNS. There are VMVersion and hostVersion which compared with this Version. So I added another explicite NCVersion to ensure AKS-Swift usage. However, after double checking every usage of Version, I can confirm Version meant NCVersion. I'll reuse it.
2. I'll remove NCVersion. If we want to change Version to int, we can open a separate PR to change all the existing Version.
3. ProgrammedVersion is persisted in containerstatus already.